### PR TITLE
gupnp-igd: 0.2.5 -> 1.2.0

### PR DESCRIPTION
**Pull request by amaxine**
###### Motivation for this change
Version bump. Removes the need for a compatibility patch. Upstream switched to meson.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 116118` at 86aee92e run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>9 packages marked as broken and skipped:</summary>
  <ul>
    <li>gnome3.gnome-books</li>
    <li>gnome3.gnome-documents</li>
    <li>libsForQt512.dragon</li>
    <li>libsForQt512.kamoso</li>
    <li>libsForQt512.kdenlive</li>
    <li>libsForQt514.kdenlive</li>
    <li>openmodelica</li>
    <li>termplay</li>
    <li>xmonad_log_applet</li>
  </ul>
</details>
<details>
  <summary>1 package failed to build (new failures):</summary>
  <ul>
    <li><a href="https://gist.github.com/66f21721881d778c31490f9db5972f8f">ib-controller</a></li></li>
  </ul>
</details>
<details>
  <summary>231 packages skipped due to time constraints:</summary>
  <ul>
    <li>adapta-gtk-theme</li>
    <li>almanah</li>
    <li>apostrophe</li>
    <li>aravis</li>
    <li>areca</li>
    <li>astroid</li>
    <li>aws-workspaces</li>
    <li>azureus</li>
    <li>balsa</li>
    <li>birdfont</li>
    <li>...</li>
  </ul>
</details>
<details>
  <summary>83 packages built successfully:</summary>
  <ul>
    <li><a href="https://gist.github.com/64f1998fd2615e2a880d020a18228380">audio-recorder</a></li></li>
    <li><a href="https://gist.github.com/05aa46f664546e994c67b4c21025fc0e">baresip</a></li></li>
    <li>brasero</li>
    <li><a href="https://gist.github.com/984044da112f3d3ac6c1246284b0ca01">brasero-original</a></li></li>
    <li><a href="https://gist.github.com/cdd22936b4a9294860a8872653a57fdb">byzanz</a></li></li>
    <li><a href="https://gist.github.com/01f72b8118bc5f053ecfbcf1a34c360e">cawbird</a></li></li>
    <li><a href="https://gist.github.com/74aea64db8e324254fc9fbd6c333552f">enlightenment.rage</a></li></li>
    <li><a href="https://gist.github.com/f2fbcd109395293f98d695c50b4793e0">evince</a></li> (gnome3.evince)</li>
    <li><a href="https://gist.github.com/50517a09d095c06b70f4429927e70965">farstream</a></li></li>
    <li><a href="https://gist.github.com/8a9f9ebd2eb3a0056ee653e2bb07158c">gajim</a></li></li>
    <li><a href="https://gist.github.com/5f91498eb55245fdae734f3d101c64dc">gcompris</a></li></li>
    <li><a href="https://gist.github.com/19a2add91e86f6d40f9ec3f20767523d">gmrender-resurrect</a></li></li>
    <li><a href="https://gist.github.com/20b78b1c3fdee6634d07a42332df4e34">gnome-network-displays</a></li></li>
    <li><a href="https://gist.github.com/6d67383f4ef77673367d8f17ea2579c5">gscrabble</a></li></li>
    <li><a href="https://gist.github.com/e776b5370f354b07a3b5c53bee9b5d8b">gst_all_1.gst-editing-services</a></li></li>
    <li><a href="https://gist.github.com/694e655244b9a588a8ce884164ca5b90">gst_all_1.gst-plugins-bad</a></li></li>
    <li><a href="https://gist.github.com/7ac2c9edac5ee1dcd71e09e4c484e74c">gst_all_1.gst-rtsp-server</a></li></li>
    <li><a href="https://gist.github.com/9dcd15bbbad552c76f04359af356b078">gst_all_1.gst-vaapi</a></li></li>
    <li><a href="https://gist.github.com/2a413b62be0802dc05be60daa08c320e">gupnp-igd</a></li></li>
    <li><a href="https://gist.github.com/ae010a7dea69c9f9271394abe8ca4f03">libnice</a></li></li>
    <li><a href="https://gist.github.com/247d5116f554bdffa824c1085a601ace">libsForQt5.dragon</a></li> (libsForQt515.dragon ,plasma5Packages.dragon)</li>
    <li><a href="https://gist.github.com/266bdb78c957dedfcdd0ff987cf988c8">libsForQt5.kamoso</a></li> (libsForQt515.kamoso ,plasma5Packages.kamoso)</li>
    <li><a href="https://gist.github.com/9b4f9f3aeae1f5dbf8765313851ef37c">libsForQt5.phonon-backend-gstreamer</a></li> (libsForQt515.phonon-backend-gstreamer ,plasma5Packages.phonon-backend-gstreamer)</li>
    <li><a href="https://gist.github.com/9879862e6bd08a4d859e2bdfbba8bf69">libsForQt512.phonon-backend-gstreamer</a></li></li>
    <li><a href="https://gist.github.com/4a62d38ad7144d83fdb7ec4d851751a9">libsForQt514.dragon</a></li></li>
    <li><a href="https://gist.github.com/ad7b9dde20e3437dc5475a9251a6f10e">libsForQt514.kamoso</a></li></li>
    <li><a href="https://gist.github.com/177ec7fdef4a06095a40a902e3782f6b">libsForQt514.phonon-backend-gstreamer</a></li></li>
    <li><a href="https://gist.github.com/f1bbb4f1774141ac9649a09b4e2f9293">mailnag</a></li></li>
    <li><a href="https://gist.github.com/b0ae53c639a5b491f6ee24c5382443b4">mopidy-iris</a></li></li>
    <li><a href="https://gist.github.com/a469a6c6c30f3672c4b64d26cccc491a">mopidy-local</a></li></li>
    <li><a href="https://gist.github.com/9495a28ddeb02417fee011ae59327e72">mopidy-mopify</a></li></li>
    <li><a href="https://gist.github.com/fd45fa82f8539f2dae82b45fce39221f">mopidy-mpd</a></li></li>
    <li><a href="https://gist.github.com/29a432c5249e626fe4e3cfba7a7b89b2">mopidy-mpris</a></li></li>
    <li><a href="https://gist.github.com/878474bf8b53a8770bfc89e87af92878">mopidy-musicbox-webclient</a></li></li>
    <li><a href="https://gist.github.com/d9288472f31f5954072dd4b6577769a5">mopidy-scrobbler</a></li></li>
    <li><a href="https://gist.github.com/abbcbbd5948c211fd4a6d2d18bfe519b">mopidy-somafm</a></li></li>
    <li><a href="https://gist.github.com/7b05163072c4c79132307ab05ccfae93">mopidy-soundcloud</a></li></li>
    <li><a href="https://gist.github.com/fc190bf06d2ffede33b57a1c3195ab1a">mopidy-spotify-tunigo</a></li></li>
    <li><a href="https://gist.github.com/eb8ca96e34a4158ccfd254cf84abb52f">mopidy-subidy</a></li></li>
    <li><a href="https://gist.github.com/c4d7925638aca5a34d3636cf7399de45">mopidy-tunein</a></li></li>
    <li><a href="https://gist.github.com/e8548e6bfd68e7ef419cadf2718a811b">mopidy-youtube</a></li></li>
    <li><a href="https://gist.github.com/6cdca703e3601095783f6a235ecc84b7">pantheon.elementary-camera</a></li></li>
    <li><a href="https://gist.github.com/637a6ef27d871753dfe49f7c6cd0f972">pantheon.elementary-music</a></li></li>
    <li><a href="https://gist.github.com/9342c9a83a6b43fe0e69f2589e4cf126">pantheon.elementary-videos</a></li></li>
    <li><a href="https://gist.github.com/16d5e69a62aa86554b80ce5c505ac9fc">parlatype</a></li></li>
    <li><a href="https://gist.github.com/2f84749e837591a66951ba38d63a5b90">picard</a></li></li>
    <li><a href="https://gist.github.com/90fcd47acd57ec0138522485fa6f065c">pidgin</a></li></li>
    <li><a href="https://gist.github.com/ebd92b23dc036337904e737ddfa04161">pidgin-carbons</a></li></li>
    <li><a href="https://gist.github.com/21f65e24440e6502ae425e65d0f04afd">pidgin-latex</a></li></li>
    <li><a href="https://gist.github.com/c56dfc23e05a457c16a2a2cabdda919a">pidgin-mra</a></li></li>
    <li><a href="https://gist.github.com/913e90a0d716f06553e3306f9ad980a7">pidgin-msn-pecan</a></li></li>
    <li><a href="https://gist.github.com/37ca8baf8a513278c75c39b53e55526d">pidgin-opensteamworks</a></li></li>
    <li><a href="https://gist.github.com/9f867589828231ffb26ecbc7880aceaa">pidgin-osd</a></li></li>
    <li><a href="https://gist.github.com/1caa5b6d490eb31ce0d644ad76cf6e73">pidgin-otr</a></li></li>
    <li><a href="https://gist.github.com/05e724951c7d027b337381eac4b23c8a">pidgin-sipe</a></li></li>
    <li><a href="https://gist.github.com/b5cdec527dbf72ad1a1efc1ccc7ffa1b">pidgin-skypeweb</a></li></li>
    <li><a href="https://gist.github.com/7b2acb1ea0b13c3e9dad37ff80d946f3">pidgin-window-merge</a></li></li>
    <li><a href="https://gist.github.com/3718946839763abafdf851672e9ee08d">pidgin-xmpp-receipts</a></li></li>
    <li><a href="https://gist.github.com/45457a8935a4459544942acade47e052">pithos</a></li></li>
    <li><a href="https://gist.github.com/ff41246279ca7686e3af9a693e241692">pitivi</a></li></li>
    <li><a href="https://gist.github.com/2447618480699c529a726b05782bb3b1">purple-discord</a></li></li>
    <li><a href="https://gist.github.com/1b778b917fbf6a58d4338f4b30a128f7">purple-facebook</a></li></li>
    <li><a href="https://gist.github.com/5a21e4716122eabe4629746791aac8b3">purple-hangouts</a></li></li>
    <li><a href="https://gist.github.com/f706b66a368f02c1e966ff0b66f49adc">purple-lurch</a></li></li>
    <li><a href="https://gist.github.com/444234dfc4f4545c95494b30e3bab53d">purple-matrix</a></li></li>
    <li><a href="https://gist.github.com/466719bc183f19a4b596da2b5ddd7e09">purple-slack</a></li></li>
    <li><a href="https://gist.github.com/b381d86fc7854f3d21f97db8c63f3cf3">purple-vk-plugin</a></li></li>
    <li><a href="https://gist.github.com/244f736b90d9466f0b23e6fad38432f5">purple-xmpp-http-upload</a></li></li>
    <li><a href="https://gist.github.com/3013287a783165a832d36b60043329db">python38Packages.kivy</a></li></li>
    <li><a href="https://gist.github.com/96544264358021b653b4d0c16de720db">qt-video-wlr</a></li></li>
    <li><a href="https://gist.github.com/adabeb27b66442c666a64c0b80b5b960">qutebrowser</a></li></li>
    <li><a href="https://gist.github.com/d0626ae17b052084ac773338756c4757">run-scaled</a></li></li>
    <li><a href="https://gist.github.com/7ea5ad70b65c89ce006357aaf56a75e4">skype4pidgin</a></li></li>
    <li><a href="https://gist.github.com/8ffa7e93901cac18b8f014a4f46c9e94">sound-juicer</a></li></li>
    <li><a href="https://gist.github.com/6bd9f2ff19f6fae5f1121ce216f0ab51">telegram-purple</a></li></li>
    <li><a href="https://gist.github.com/8b692d496851692918464e1a976da68c">telepathy-farstream</a></li></li>
    <li><a href="https://gist.github.com/6dad9b8ef3670e172f230da433be525a">telepathy-gabble</a></li></li>
    <li><a href="https://gist.github.com/0c293e38a3b8d278d8ea9437255a9d6e">telepathy-haze</a></li></li>
    <li><a href="https://gist.github.com/ae8c9335fbdab40eaaba00479f1e5f55">toxprpl</a></li></li>
    <li><a href="https://gist.github.com/3680c9ee7c387559ad7ad2ea2b5d3c0d">vokoscreen-ng</a></li></li>
    <li><a href="https://gist.github.com/6d1cae9bab05b19a9c160f72116aef9b">xfce.parole</a></li></li>
    <li><a href="https://gist.github.com/7a1c9ee6c51d7805bebd0fec25f911b4">xpra</a></li></li>
    <li><a href="https://gist.github.com/5575fb9c6e5e41175f482f3a768979f3">yaxg</a></li></li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/maintainers-missing.md">maintainers-missing</a>

Package does not have a maintainer. Consider adding yourself?

Near pkgs/development/libraries/gupnp-igd/default.nix:55:3:
```
   |
55 |   meta = with lib; {
   |   ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md">unclear-gpl</a>

`lgpl21` is a deprecated license, check if project uses `lgpl21Plus` or `lgpl21Only` and change `meta.license` accordingly.

Near pkgs/development/libraries/gupnp-igd/default.nix:58:5:
```
   |
58 |     license = licenses.lgpl21;
   |     ^
```
</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 116118` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
9 packages marked as broken and skipped:
</summary>
<ul>
<li>gnome3.gnome-books</li>
<li>gnome3.gnome-documents</li>
<li>libsForQt512.dragon</li>
<li>libsForQt512.kamoso</li>
<li>libsForQt512.kdenlive</li>
<li>libsForQt514.kdenlive</li>
<li>openmodelica</li>
<li>termplay</li>
<li>xmonad_log_applet</li>
</ul>
</details>
<details>

  <summary>5 packages failed to build and already failed to build on hydra master:</summary>
  <ul>
<li>fractal: log was empty</li>
<li>gnome-podcasts: log was empty</li>
<li>gnunet-gtk: log was empty</li>
<li>gtkd: log was empty</li>
<li>tilix: log was empty</li>
  </ul>
</details>
<details>

  <summary>9 packages failed to build and are new build failures:</summary>
  <ul>
<li>citrix_workspace: log was empty</li>
<li>(citrix_workspace_21_01_0): log was empty</li>
<li>citrix_workspace_20_04_0: log was empty</li>
<li>citrix_workspace_20_06_0: log was empty</li>
<li>citrix_workspace_20_09_0: log was empty</li>
<li>citrix_workspace_20_10_0: log was empty</li>
<li>citrix_workspace_20_12_0: log was empty</li>
<li>ib-controller: log was empty</li>
<li>luakit: log was empty</li>
  </ul>
</details>
<details>
<summary>
297 packages built:
</summary>
<ul>
<li>adapta-gtk-theme</li>
<li>almanah</li>
<li>apostrophe</li>
<li>aravis</li>
<li>areca</li>
<li>astroid</li>
<li>audio-recorder</li>
<li>aws-workspaces</li>
<li>azureus</li>
<li>balsa</li>
<li>baresip</li>
<li>birdfont</li>
<li>bookworm</li>
<li>brasero</li>
<li>brasero-original</li>
<li>bubblemail</li>
<li>byzanz</li>
<li>caerbannog</li>
<li>calls</li>
<li>cawbird</li>
<li>chrome-gnome-shell</li>
<li>cinnamon.cinnamon-common</li>
<li>cinnamon.cinnamon-control-center</li>
<li>cinnamon.cinnamon-gsettings-overrides</li>
<li>cinnamon.cinnamon-screensaver</li>
<li>claws-mail-gtk3</li>
<li>cog</li>
<li>denemo</li>
<li>dropbox-cli</li>
<li>eclipse-mat</li>
<li>eclipses.eclipse-committers</li>
<li>eclipses.eclipse-cpp</li>
<li>eclipses.eclipse-java</li>
<li>eclipses.eclipse-modeling</li>
<li>eclipses.eclipse-platform</li>
<li>eclipses.eclipse-rcp</li>
<li>eclipses.eclipse-rust</li>
<li>eclipses.eclipse-scala-sdk</li>
<li>eclipses.eclipse-sdk</li>
<li>elementary-planner</li>
<li>empathy (gnome3.empathy)</li>
<li>enlightenment.rage</li>
<li>eolie</li>
<li>ephemeral</li>
<li>epiphany (gnome3.epiphany)</li>
<li>evince (gnome3.evince)</li>
<li>evolution</li>
<li>evolution-data-server (gnome3.evolution-data-server ,gnome3.evolution_data_server)</li>
<li>evolution-ews</li>
<li>evolutionWithPlugins</li>
<li>farstream</li>
<li>fcitx5-gtk</li>
<li>fcitx5-with-addons</li>
<li>feedreader</li>
<li>feeds</li>
<li>folks (gnome3.folks)</li>
<li>formiko</li>
<li>gajim</li>
<li>gcompris</li>
<li>gfbgraph (gnome3.gfbgraph)</li>
<li>glade (gnome3.glade)</li>
<li>glom</li>
<li>gmrender-resurrect</li>
<li>gnome-builder</li>
<li>gnome-keysign</li>
<li>gnome-network-displays</li>
<li>gnome-online-accounts (gnome3.gnome-online-accounts ,gnome3.gnome_online_accounts)</li>
<li>gnome-passwordsafe</li>
<li>gnome-photos (gnome3.gnome-photos)</li>
<li>gnome-recipes</li>
<li>gnome-tour</li>
<li>gnome3.bijiben (gnome3.gnome-notes)</li>
<li>gnome3.cheese</li>
<li>gnome3.devhelp</li>
<li>gnome3.file-roller</li>
<li>gnome3.geary</li>
<li>gnome3.gnome-applets</li>
<li>gnome3.gnome-boxes</li>
<li>gnome3.gnome-calendar</li>
<li>gnome3.gnome-clocks</li>
<li>gnome3.gnome-contacts</li>
<li>gnome3.gnome-control-center (gnome3.gnome_control_center)</li>
<li>gnome3.gnome-flashback</li>
<li>gnome3.gnome-initial-setup</li>
<li>gnome3.gnome-maps</li>
<li>gnome3.gnome-music</li>
<li>gnome3.gnome-online-miners</li>
<li>gnome3.gnome-panel</li>
<li>gnome3.gnome-screenshot</li>
<li>gnome3.gnome-session (gnome3.gnome_session)</li>
<li>gnome3.gnome-shell (gnome3.gnome_shell)</li>
<li>gnome3.gnome-software</li>
<li>gnome3.gnome-sound-recorder</li>
<li>gnome3.gnome-terminal (gnome3.gnome_terminal)</li>
<li>gnome3.gnome-todo</li>
<li>gnome3.gnome-tweaks (gnome3.gnome-tweak-tool)</li>
<li>gnome3.gnome-user-share</li>
<li>grilo-plugins (gnome3.grilo-plugins)</li>
<li>gnome3.gvfs</li>
<li>libgdata (gnome3.libgdata)</li>
<li>libgepub (gnome3.libgepub)</li>
<li>libzapojit (gnome3.libzapojit)</li>
<li>gnome3.nautilus</li>
<li>gnome3.nautilus-python</li>
<li>gnome3.polari</li>
<li>gnome3.pomodoro</li>
<li>gnome3.rygel</li>
<li>gnome3.seahorse</li>
<li>shotwell (gnome3.shotwell)</li>
<li>gnome3.sushi</li>
<li>gnome3.totem</li>
<li>tracker-miners (gnome3.tracker-miners)</li>
<li>webkitgtk (gnome3.webkitgtk)</li>
<li>yelp (gnome3.yelp)</li>
<li>gnomeExtensions.easyScreenCast</li>
<li>gnomeExtensions.gsconnect</li>
<li>gnomeExtensions.night-theme-switcher</li>
<li>gnucash</li>
<li>gnvim</li>
<li>gnvim-unwrapped</li>
<li>gscrabble</li>
<li>gst_all_1.gst-editing-services</li>
<li>gst_all_1.gst-plugins-bad</li>
<li>gst_all_1.gst-rtsp-server</li>
<li>gst_all_1.gst-vaapi</li>
<li>gthumb</li>
<li>gtk4</li>
<li>gupnp-igd</li>
<li>haskellPackages.reflex-dom</li>
<li>haskellPackages.webkit2gtk3-javascriptcore</li>
<li>ipscan</li>
<li>kicad-small</li>
<li>kicad-unstable-small</li>
<li>libhandy</li>
<li>libnice</li>
<li>libreoffice-fresh</li>
<li>libreoffice-fresh-unwrapped</li>
<li>libreoffice-qt</li>
<li>libreoffice-still</li>
<li>libreoffice-unwrapped (libreoffice-still-unwrapped)</li>
<li>libsForQt5.dragon (libsForQt515.dragon ,plasma5Packages.dragon)</li>
<li>libsForQt5.kamoso (libsForQt515.kamoso ,plasma5Packages.kamoso)</li>
<li>libsForQt5.kdenlive (libsForQt515.kdenlive ,plasma5Packages.kdenlive)</li>
<li>libsForQt5.phonon-backend-gstreamer (libsForQt515.phonon-backend-gstreamer ,plasma5Packages.phonon-backend-gstreamer)</li>
<li>libsForQt5.telepathy (libsForQt515.telepathy ,plasma5Packages.telepathy)</li>
<li>libsForQt512.phonon-backend-gstreamer</li>
<li>libsForQt512.telepathy</li>
<li>libsForQt514.dragon</li>
<li>libsForQt514.kamoso</li>
<li>libsForQt514.phonon-backend-gstreamer</li>
<li>libsForQt514.telepathy</li>
<li>liferea</li>
<li>lispPackages.cl-webkit2</li>
<li>lispPackages.nyxt</li>
<li>lollypop</li>
<li>lutris</li>
<li>lutris-free</li>
<li>lutris-unwrapped</li>
<li>mailnag</li>
<li>mailnagWithPlugins</li>
<li>mapmap</li>
<li>marker</li>
<li>mate.atril</li>
<li>mate.mate-user-guide</li>
<li>mavproxy</li>
<li>meteo</li>
<li>midori-unwrapped</li>
<li>mmex</li>
<li>mopidy-iris</li>
<li>mopidy-local</li>
<li>mopidy-mopify</li>
<li>mopidy-mpd</li>
<li>mopidy-mpris</li>
<li>mopidy-musicbox-webclient</li>
<li>mopidy-scrobbler</li>
<li>mopidy-somafm</li>
<li>mopidy-soundcloud</li>
<li>mopidy-spotify-tunigo</li>
<li>mopidy-subidy</li>
<li>mopidy-tunein</li>
<li>mopidy-youtube</li>
<li>nasc</li>
<li>newsflash</li>
<li>nheko</li>
<li>notes-up</li>
<li>nyxt</li>
<li>odpdown</li>
<li>onboard</li>
<li>osmo</li>
<li>pantheon.elementary-calendar</li>
<li>pantheon.elementary-camera</li>
<li>pantheon.elementary-capnet-assist</li>
<li>pantheon.elementary-code</li>
<li>pantheon.elementary-greeter</li>
<li>pantheon.elementary-gsettings-schemas</li>
<li>pantheon.elementary-music</li>
<li>pantheon.elementary-onboarding</li>
<li>pantheon.elementary-photos</li>
<li>pantheon.elementary-session-settings</li>
<li>pantheon.elementary-videos</li>
<li>pantheon.extra-elementary-contracts</li>
<li>pantheon.notes-up</li>
<li>pantheon.switchboard-plug-a11y</li>
<li>pantheon.switchboard-plug-onlineaccounts</li>
<li>pantheon.switchboard-with-plugs</li>
<li>pantheon.wingpanel-applications-menu</li>
<li>pantheon.wingpanel-indicator-datetime</li>
<li>pantheon.wingpanel-with-indicators</li>
<li>parlatype</li>
<li>pdfpc</li>
<li>picard</li>
<li>pidgin</li>
<li>pidgin-carbons</li>
<li>pidgin-latex</li>
<li>pidgin-mra</li>
<li>pidgin-msn-pecan</li>
<li>pidgin-opensteamworks</li>
<li>pidgin-osd</li>
<li>pidgin-otr</li>
<li>pidgin-sipe</li>
<li>pidgin-skypeweb</li>
<li>pidgin-window-merge</li>
<li>pidgin-xmpp-receipts</li>
<li>pipelight</li>
<li>pithos</li>
<li>pitivi</li>
<li>playonlinux</li>
<li>portfolio</li>
<li>pragha</li>
<li>printrun</li>
<li>pulseeffects-legacy</li>
<li>pulseeffects-pw</li>
<li>purple-discord</li>
<li>purple-facebook</li>
<li>purple-hangouts</li>
<li>purple-lurch</li>
<li>purple-matrix</li>
<li>purple-plugin-pack</li>
<li>purple-slack</li>
<li>purple-vk-plugin</li>
<li>purple-xmpp-http-upload</li>
<li>python38Packages.kicad</li>
<li>python38Packages.kivy</li>
<li>python38Packages.wxPython_4_0</li>
<li>python38Packages.wxPython_4_1</li>
<li>python39Packages.kicad</li>
<li>python39Packages.kivy</li>
<li>python39Packages.wxPython_4_0</li>
<li>python39Packages.wxPython_4_1</li>
<li>pytrainer</li>
<li>qgroundcontrol</li>
<li>qt-video-wlr</li>
<li>quilter</li>
<li>quisk</li>
<li>quodlibet</li>
<li>quodlibet-full</li>
<li>quodlibet-without-gst-plugins</li>
<li>quodlibet-xine</li>
<li>quodlibet-xine-full</li>
<li>qutebrowser</li>
<li>radiotray-ng</li>
<li>rednotebook</li>
<li>remmina</li>
<li>run-scaled</li>
<li>sayonara</li>
<li>setzer</li>
<li>shortwave</li>
<li>skype4pidgin</li>
<li>sound-juicer</li>
<li>sparkleshare</li>
<li>subtitleeditor</li>
<li>surf</li>
<li>surf-display</li>
<li>swt</li>
<li>swt_jdk8</li>
<li>tangram</li>
<li>telegram-purple</li>
<li>telepathy-farstream</li>
<li>telepathy-gabble</li>
<li>telepathy-haze</li>
<li>tootle</li>
<li>toxprpl</li>
<li>tunefish</li>
<li>tuxguitar</li>
<li>ulauncher</li>
<li>unoconv</li>
<li>vala-language-server</li>
<li>vimb-unwrapped</li>
<li>vocal</li>
<li>vokoscreen-ng</li>
<li>vuze</li>
<li>webkit2-sharp</li>
<li>xfce.parole</li>
<li>xiphos</li>
<li>xplayer</li>
<li>xpra</li>
<li>yaxg</li>
</ul>
</details>

The following issues got detected with the above build packages.
Please fix at least the ones listed with your changed packages:

<details>
adapta-gtk-theme:
<details>


warning: unclear-gpl
`gpl2` is a deprecated license, check if project uses `gpl2Plus` or `gpl2Only` and change `meta.license` accordingly.

Near pkgs/data/themes/adapta/default.nix:45:5:

```
   |
45 |     license = with licenses; [ gpl2 cc-by-sa-30 ];
   |     ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md
</details>
</details>

**Comment by maxeaubrey** [comment]
Citrix failures are from needing to prefetch packages, same for `ib-controller`, haven't tested if they work when that's done.
I'll look into what's with `luakit`, my build machine ran out of space while building webkitgtk so I haven't gotten to the log.

> adapta-gtk-theme
Don't think that's related :sweat_smile: but I fixed the license for `gupnp-igd`. Not adding myself as a maintainer tho.

File: pkgs/development/libraries/gupnp-igd/default.nix [hunk 1]
```diff
@@ -62,7 +55,7 @@ stdenv.mkDerivation rec {
   meta = with lib; {
     description = "Library to handle UPnP IGD port mapping";
     homepage = "http://www.gupnp.org/";
-    license = licenses.lgpl21;
+    license = licenses.lgpl21Only;
```
Comment (jtojnar) [inline pkgs/development/libraries/gupnp-igd/default.nix:58]: https://gitlab.gnome.org/GNOME/gupnp-igd/-/blob/d0ecd7812b5c99e5f31fba0a707724c1dce1f94a/libgupnp-igd/gupnp-simple-igd.c#L9-11

**Comment by jtojnar** [comment]
Diffing the outputs, `Requires: gupnp-1.2` in the pc file changed to `Requires.private: glib-2.0 >=  2.38, gobject-2.0 >=  2.38, gupnp-1.2 >=  1.2.0, gssdp-1.2 >=  1.2.0, gthread-2.0`.

Looking at the includes in the headers, I would expect `glib` and `gobject` in `Requires`. But it probably will not cause issues for us because anything linking against gupnp will probably also link against glib.

**Review by jtojnar — APPROVED** [review]
thanks

**Pull request merged**